### PR TITLE
Login 119 v2

### PR DESCRIPTION
**Pull request by JeffAbrahamson**
Branch cleaned and ready for PR.

### CODE REVIEW

**Comment by JeffAbrahamson** [comment]
Closes #51 and #119.

**Review by Shriukan33 — COMMENTED** [review]
Other than my question, LGTM. 

Thanks for fixing the failing test on TBv2 !

File: transport_nantes/authentication/views.py [hunk 1]
```diff
@@ -138,6 +138,8 @@ def send_activation(request, user, is_new, remember_me):
     })
     if hasattr(settings, 'ROLE') and settings.ROLE in ['staging', 'production']:
         user.email_user(subject, message)
+    elif hasattr(settings, 'ROLE') and settings.ROLE == 'test':
```
[the remark this answers is not in the document]
Comment (JeffAbrahamson) [reply transport_nantes/authentication/views.py]: That's not at head of the branch.  At HEAD, it switches based on `TEST_MODE`, which is defined if you run the tests via the Makefile.

[the remark this answers is not in the document]
Comment (Shriukan33) [reply transport_nantes/authentication/views.py]: So tests should be run from make now ? `make test` ?

[the remark this answers is not in the document]
Comment (JeffAbrahamson) [reply transport_nantes/authentication/views.py]: That's the goal of implementing abstractions and automation: to codify in _code_ how best to do things without forgetting details.

**Comment by Shriukan33** [comment]
Just noticed that the wording of account_activation_email is off, 

```
Bonjour,

Si toutefois vous n'avez pas demandé la création d'un compte, vous
pouvez simplement supprimer ce mél.

{{ uri }}

{{ scheme }}://{{ host }}{% url 'authentication:activate' token=token %}
```

Starts with "Si toutefois", which sounds like it should have a sentence before that.

**Pull request merged**